CMake/SlicerMacroBuildApplication.cmake

```diff
@@ -540,7 +540,16 @@ macro(slicerMacroBuildApplication)
       # Define list of extra 'application to launch' to associate with the launcher
       # within the build tree
       set(extraApplicationToLaunchListForBuildTree)
-      if(EXISTS "${QT_DESIGNER_EXECUTABLE}")
+
+      if(${Slicer_REQUIRED_QT_VERSION} VERSION_GREATER_EQUAL 5)
+        find_package(Qt5 COMPONENTS Designer QUIET)
+        if(Qt5Designer_FOUND)
+          # This is not perfect, but I cannot find a way for the CMake configuration to provide me with a path to the exe
```

Comment:
why is it not perfect?

Reply:
I'd prefer it if find_package(Qt5...) populated a variable like Qt5Designer_EXECUTABLE or similar.

That way it's up to Qt5 to tell us where it is, instead of us guessing (aka, if they move it, change it, etc...)

Reply:
Agreed. In the mean time, that will do it.

May be you could suggest a path to upstream Qt to add `Qt5::designertool` target similar to `Qt5::qmake`, etc ...

Reply:
OK, then just add a note that explains this (that it is supposed to be provided by CMake, but implement this here for now).